Components/Buttons/buttons.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" href="../../Assets/light.png">
+    <link rel="stylesheet" href="../../utilsCSS/getting-started.css" />
+    <link rel="stylesheet" href="./Buttons.css" />
+    <script src="https://kit.fontawesome.com/e06444868d.js" crossorigin="anonymous"></script>
+    <title>Buttons | Avid</title>
+</head>
+
+<body>
+    <header class="header">
+        <div class="logo">
+            <a href="/index.html">
+                <img class="logo" src="../../Assets/light.png" />
+            </a>
+        </div>
+
+        <nav class="navbar">
+            <ul>
+                <a href="/index.html">
+                    <li class="highlight">Home</li>
+                </a>
+                <a href="./getting-started.html">
+                    <li class="highlight">Documentation</li>
+                </a>
+            </ul>
+        </nav>
+    </header>
+    <section class="documentation">
+        <aside class="sidebar">
+            <div class="list-topic">
+                <h4 class="highlight">Get Started</h4>
+            </div>
+            <ul>
+                <li><a href="../../getting-started.html">Introduction</a></li>
+                <!-- <li><a href="../Colors/colors.html">Colors</a></li>
+          <li><a href="../Typography/typography.html">Typography</a></li> -->
+            </ul>
+
+            <div class="list-topic">
+                <h4 class="highlight">Components</h4>
+            </div>
+            <ul>
+                <li><a href="../Cards/card.html">Card</a></li>
+                <!-- <li><a href="./Components/Avatar/avatar.html">Avatar</a></li>
```

Comment:
another suggestion is to remove commented code

Reply:
noted.